### PR TITLE
Models

### PR DESCRIPTION
**Pull request by TravisMWise**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -36,18 +36,36 @@ const main = async () => {
...
 
-  manager.addObjects([myRayman, myCow]);
+
+  // manager.addObjects([myRayman, myCow]);
+  // manager.addObjects([myAsteroid0]);
+  manager.addObjects([myAsteroid1]);
+  // manager.addObjects([starwars]);
+  // manager.addObjects([ufo]);
+
+  myAsteroid1.physics.angularVelocity = new Vector3(0, 30, 0);
```
Comment (Armintron) [inline src/index.js:64]: Nit-pick: this should happen before you add the object to the gamemanager

File: src/index.js [hunk 2]
```diff
@@ -36,18 +36,36 @@ const main = async () => {
...
+  const starwars = new GameObject(manager.modelList[4], new Physics());
+  const ufo = new GameObject(manager.modelList[5], new Physics());
 
-  manager.addObjects([myRayman, myCow]);
+
+  // manager.addObjects([myRayman, myCow]);
+  // manager.addObjects([myAsteroid0]);
+  manager.addObjects([myAsteroid1]);
+  // manager.addObjects([starwars]);
+  // manager.addObjects([ufo]);
```
Comment (Armintron) [inline src/index.js:62]: Commented code can be deleted

File: src/index.js [hunk 3]
```diff
@@ -36,18 +36,36 @@ const main = async () => {
...
+  const myAsteroid0 = new GameObject(manager.modelList[2], new Physics());
+  const myAsteroid1 = new GameObject(manager.modelList[3], new Physics());
+  const starwars = new GameObject(manager.modelList[4], new Physics());
+  const ufo = new GameObject(manager.modelList[5], new Physics());
 
-  manager.addObjects([myRayman, myCow]);
+
+  // manager.addObjects([myRayman, myCow]);
+  // manager.addObjects([myAsteroid0]);
+  manager.addObjects([myAsteroid1]);
```
Comment (Armintron) [inline src/index.js:60]: Use `addObject` function

File: src/index.js [hunk 4]
```diff
@@ -36,18 +36,36 @@ const main = async () => {
...
+  // manager.addObjects([starwars]);
+  // manager.addObjects([ufo]);
+
+  myAsteroid1.physics.angularVelocity = new Vector3(0, 30, 0);
 
   myCow.physics.angularVelocity = new Vector3(10, 10, 10);
 
-  const raymanModelExtents = manager.modelList[0].getModelExtent();
+  const raymanModelExtents = manager.modelList[3].getModelExtent();
```
Comment (Armintron) [inline src/index.js:68]: Rename variable since it's no longer Rayman extents

**Review by Armintron — CHANGES_REQUESTED** [review]

**Comment by ctcuff** [comment]
Overall looks good but I'd like to possibly wait on #10 to be addressed first. Having to access models from the manager via index will get cumbersome when we have multiple models to reference

**Review by Armintron — APPROVED** [review]

**Comment by Armintron** [comment]
> Overall looks good but I'd like to possibly wait on #10 to be addressed first. Having to access models from the manager via index will get cumbersome when we have multiple models to reference

This has been changed

**Pull request merged**